src/app/containers/ArticleTimestamp/index.stories.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import ArticleTimestamp from '.';
+import { timestampGenerator } from '../Timestamp/helpers/testHelpers';
+
+const fiveHoursAgo = timestampGenerator({
+  hours: 5,
+});
+
+storiesOf('ArticleTimestamp', module)
+  .add('default', () => (
```

Comment:
Can we please have all 8 scenarios for the Article Container as stories here? This will give us all visibility of the various formats that the Article Container can have. 

You can see the scenarios in the description here: https://github.com/bbc/simorgh/pull/1619 Or by just looking at the unit tests.

Reply:
@Bopchy Thanks for adding all of these scenarios. It's great to have visibility of the variants of this container in Storybook.